src/public/interfaces.ts

```diff
@@ -384,6 +384,16 @@ export interface OpenConversationRequest {
   */
   conversationId?: string;
 
+  /**
+   * The Id of the channel. This is optional and should be specified whenever a conversation is started or opened in a personal app scope
+   */
+  threadId?: string;
+
+  /**
+   * The Id of the tab. This is optional and should be specified whenever a conversation is started in a personal scope
+   */
+  entityId: string;
```

Comment:
If it’s optional should this be “entityId?”, like threadId above

Reply:
Its not optional. We changed it. Forgot to update the comment. Done